nerfstudio/data/datamanagers.py

```diff
@@ -240,12 +244,18 @@ class VanillaDataManagerConfig(InstantiateConfig):
     """Specifies the dataparser used to unpack the data."""
     train_num_rays_per_batch: int = 1024
     """Number of rays per batch to use per training iteration."""
-    train_num_images_to_sample_from: int = -1
+    train_num_images_to_sample_from: int = 300
```

Comment:
I'm worried that it may not be intuitive if the default behavior doesn't used all of the training data. Perhaps instead we could default use all the images, but print a warning to the user if they are using more that 500 images that they can specify a subset.

Reply:
OK sounds good. I will revert the changes to default values in this PR, so that the additions will be opt-in instead.